mails/themes/classic/components/footer.html.twig

```diff
@@ -0,0 +1,8 @@
+            <tr>
```

Comment:
indentation

Reply:
0 spaces as the base? 2 spaces for each level?

Reply:
0 spaces as the base